CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v5.7.1
```

Comment:
Since this version is unreleased, can you also add this to the version so that there is no confusion.  Something like `v5.7.1 (Unreleased)`

Reply:
All right!